docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,44 +1,99 @@
 [[connect-to-elasticsearch]]
-== Connect Kibana with Elasticsearch
+== Adding data
```

Comment:
```suggestion
== Add data
```

Reply:
I'll make that change when I change the other titles in the section.